changelog/unreleased/issue-1863

```diff
@@ -0,0 +1,6 @@
+Bugfix: Report correct number of directories processed by backup
+
+We have fixed the directory statistics calculation to report the actual number
+of processed directories instead of always zero.
```

Comment:
Keeping things neutral, I think adjusting this to the following would be a good idea:

~~~
The directory statistics calculation was fixed to report the actual number
of processed directories instead of always zero.
~~~

Reply:
Is changed.